plugins-mu/hm-stack/hm-stack.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
This should be in `plugins`, there's no such thing as plugins-mu. It's not meant to be "a plugins directory" it's just literally "a folder of plugins", all plugins in the existing directory are programmatically loaded.

Reply:
There _is_ `plugins-mu` in the `beta` branch, I didn't make that up...

I'd actually meant to ask @roborourke this in our call this morning and of course forgot 🤦‍♂️ I wasn't 100% sure where to put this but it due to a couple of things it made sense to me to put it in a folder that effectively follows the "must-use" paradgim. Obviously totally open to moving it but this is why I thought this might be an OK place for it.

1. All of the plugins in `plugins` are external plugins that are submoduled in and appear to be selectable as to whether or not they're loaded.
2. This functionality is necessary to the Cloud Plugin running and will effectively be required by the application to load the Cloud page (surely it _can_ load without it, but afaik it will be on every platform site). This shouldn't be allowed to be turned off as it's tightly coupled to the app so shouldn't follow the platform plugin paradigm.

I chose to give it plugin headers and make it more usable in case we wanted to move it or use it on an existing non-platform site as a fall-back but they're more necessary.

Reply:
Sorry for the confusion there @joehoyle @mikeselander. In the `beta` branch I added a `plugins-mu` folder as the platform code had some performance tweaks in it, specifically removing the custom fields metabox. The intention was to add all the WP core mods we want to enforce in there. Some of the items in the plugins folder we will force loading of on production at least like the caching plugins & S3. Because `hm-platform` isn't a typical plugin we don't necessarily need to follow the standard WP paradigm.

Maybe we can rename the `plugins-mu` I added to something else like `mods`.

Reply:
We already have `lib` for this purpose. Everything in hm-platform (so far atleast) is activated via code, so that's why I don't want see the plugins / plugins-mu split. In most cases though I think its better to avoid using `lib` and instead make as much as a standalone thing that lives in `plugins`.

Reply:
> This functionality is necessary to the Cloud Plugin running and will effectively be required by the application to load the Cloud page (surely it can load without it, but afaik it will be on every platform site). This shouldn't be allowed to be turned off as it's tightly coupled to the app so shouldn't follow the platform plugin paradigm.

For now I'd put all this in the a HM Cloud Admin plugin, or something like that to keep it all encapsulated until there comes a time to make it shared for all plugins on the HM Platform.

Reply:
@joehoyle we're building a unified admin for platform here so there's no separate plugin. In terms of the react app the components there will be shared with the EK page and other future pages.

Agree regarding the plugins/plugins-mu thing - @mikeselander if you move it into the `plugins` folder and then I'll refactor the `plugins-mu` folder into `lib`.